### DISCUSSION TITLE
Email verification not working

### DISCUSSION THREAD

**Discussion by mmoser18** [community/community#47252] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I am in the process of migrating to a new email provider. For that - as a first step before deleting the old email address from my profile - I want to add the new email to my account (both addresses are currently operational for a while). 

I am then receiving an email with the green button to verify my new email address (so the new email is obviously working), but when I click onto that button in the browser I always only get an error "No email to verify" (or similar). 

I tried that several times until I got "You have exceeded our email verification rate limit. "

Any idea anyone what's wrong here??? Why does that email verification not work? 

I currently have the new email configured such that it redirects/forwards all emails to the old address. Could that cause the issue? If so, why could that be an issue?

**Comment by alex-oos** [comment]
me too

**Comment by calmwaves111** [comment]
+1，I don't know what the problem is

**Comment by mokadoka** [comment]
Since you're getting an error even though the verification email is received, it could be related to how the email checker handles forwarded emails. Try turning off the forwarding feature temporarily and then attempt the verification process again. Sometimes, you can use an [email checker](https://mailtester.ninja/) if you have issues with the way forwarded emails are processed. If that doesn't work, contact customer support!

**Comment by Tatendamarimo** [comment]
It sounds like you’re having trouble verifying your new email. Here are some quick things to check:

- Disable Forwarding: Temporarily turn off email forwarding from your new address to the old one. This might be causing the issue.
- Wait: You’ve hit the email verification rate limit. Give it a few hours before trying again.
- Clear Cache: Clear your browser’s cache and cookies, or try a different browser/incognito mode.
- Check Spam: Make sure the verification email isn’t in your spam folder.
- Contact Support: If none of this works, reach out to your email provider’s support team.

**Comment by JamesHutchison** [comment]
Seeing this trying to onboard someone to the company. They clicked 3 times. This is very inconvenient.

**Comment by mmoss** [comment]
> Wait: You’ve hit the email verification rate limit. Give it a few hours before trying again.

This is a kind of wild limit, when many of us use custom domains and need to resend emails frequently due to configuration issues.
